pkg/cli/cli.go

```diff
@@ -107,9 +107,12 @@ func NewDefaultOcCommand(in io.Reader, out, errout io.Writer) *cobra.Command {
 	cmdPathPieces := os.Args[1:]
 	pluginHandler := kubecmd.NewDefaultPluginHandler(plugin.ValidPluginFilenamePrefixes)
 
+	// TODO: this should be imported from kubectl
```

Comment:
@ardaguclu we need to expose this in kubectl so we can consume it here. 

Can you please review this commit carefully the shadowing to make sure I have not missed anything?

Reply:
> we need to expose this in kubectl so we can consume it here.

ah, I missed that. I'll open a PR to fix that in next bump.

> Can you please review this shadowing commit carefully to make sure I have not missed anything?

I compared the changes on upstream and there seems to be nothing missed. Thanks.